cli/test_execution_tail.bats

```diff
@@ -0,0 +1,80 @@
+
+load 'test_helpers/bats-support/load'
+load 'test_helpers/bats-assert/load'
+
+setup() {
+	sudo crudini --set /etc/st2/st2.conf actionrunner stream_output True
+	[[ "$?" -eq 0 ]]
+
+	sudo st2ctl restart
+	[[ "$?" -eq 0 ]]
+
+	sudo cp -r /usr/share/doc/st2/examples/ /opt/stackstorm/packs/
+	[[ "$?" -eq 0 ]]
+	[[ -d /opt/stackstorm/packs/examples ]]
+
+	st2 run packs.setup_virtualenv packs=examples -j | grep -q "$STATUS_SUCCESS"
+	[[ "$?" -eq 0 ]]
+
+	st2-register-content --register-pack /opt/stackstorm/packs/examples/ --register-all
+	[[ "$?" -eq 0 ]]
+}
+
+teardown() {
+	if [[ -d /opt/stackstorm/packs/examples ]]; then
+		st2 run packs.uninstall packs=examples
+	fi
+	[[ ! -d /opt/stackstorm/packs/examples ]]
+}
+
+@test "st2 execution tail works correctly for simple actions" {
+	run eval "st2 run examples.python_runner_print_to_stdout_and_stderr count=10 sleep_delay=1 -a | grep 'st2 execution tail' | sed 's/ st2 execution tail//'"
```

Comment:
Wonder if we should just ``--json`` flag and ``jq`` here? In theory, it would be a bit robust and wouldn't break if default user-friendly CLI output format changes.

Reply:
Yep, that would be better. There are plenty of places we can do that in the BATS tests.

Reply:
Done.